README.md

```diff
@@ -211,14 +211,7 @@ To detect if a file is updated, added or deleted:
 
 ```ruby
 FileWatcher.new(['README.rdoc']).watch do |filename, event|
-  case event
```

Comment:
It's a stupid example. Nevertheless it is also a nice and simple full example of the API. I'd like to keep this example.